misk-hibernate-testing/src/main/kotlin/misk/jdbc/VitessScaleSafetyChecks.kt

```diff
@@ -29,6 +29,7 @@ import java.sql.SQLException
 import java.sql.SQLSyntaxErrorException
 import java.sql.Timestamp
 import java.util.Locale
+import java.util.Stack
```

Comment:
`java.util.Stack` is legacy and no longer recommended. You don't need thread safety here so you should use `java.util.ArrayDeque` instead.

https://stackoverflow.com/questions/47933459/when-should-i-use-java-util-stack-vs-my-own-implementation

Reply:
Done